<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitXMLReporter -->
<testsuite ignored="0" hostname="LAPTOP-OCCGCDND" failures="1" tests="3" name="Test" time="16.792" errors="0" timestamp="2024-08-01T01:29:41 IST">
  <testcase classname="seleniumjava.OrangeHRM" name="testLogo" time="0.085"/>
  <testcase classname="seleniumjava.OrangeHRM" name="testAppUrl" time="0.036">
    <failure type="java.lang.AssertionError" message="expected [https://opensource-demo.orangehrmlive.com/web/index.pip/auth/login] but found [https://opensource-demo.orangehrmlive.com/web/index.php/auth/login]">
      <![CDATA[java.lang.AssertionError: expected [https://opensource-demo.orangehrmlive.com/web/index.pip/auth/login] but found [https://opensource-demo.orangehrmlive.com/web/index.php/auth/login]
at seleniumjava.OrangeHRM.testAppUrl(OrangeHRM.java:42)
at java.base/java.util.ArrayList.forEach(ArrayList.java:1511)
... Removed 31 stack frames]]>
    </failure>
  </testcase> <!-- testAppUrl -->
  <testcase classname="seleniumjava.OrangeHRM" name="testHomepageTitle" time="0.0">
    <skipped/>
  </testcase> <!-- testHomepageTitle -->
</testsuite> <!-- Test -->
